lib/exmeralda_web/components/layouts/root.html.heex

```diff
@@ -10,6 +10,16 @@
     <link phx-track-static rel="stylesheet" href={~p"/assets/app.css"} />
     <script defer phx-track-static type="text/javascript" src={~p"/assets/app.js"}>
     </script>
+    <meta name="title" content="Ask Exmeralda" />
+    <meta name="description" content="Choose a library, and ask me anything about it." />
+    <meta property="og:site_name" content="Exmeralda" />
+    <meta property="og:title" content="Ask Exmeralda" />
+    <meta property="og:url" content={@conn.request_path} />
```

Comment:
Should this be the root ULR? How do I get that?

Reply:
```suggestion
    <meta property="og:url" content={ExmeraldaWeb.Endpoint.url()} />
```
I think that should work

Reply:
Worked, thanks!